Script/test-podspec.sh

```diff
@@ -64,18 +80,34 @@ rm -rf $PROJECT_NAME
 mkdir -p $PROJECT_NAME && cd $PROJECT_NAME
 # Create a new Xcode project.
 swift package init
-swift package generate-xcodeproj
+
+# Use Xcodegen to generate the project.
+echo "
+name: $PROJECT_NAME
+options:
+  bundleIdPrefix: $PROJECT_NAME
+targets:
+  $PROJECT_NAME:
+    type: framework
+    sources: Sources
+    platform: tvOS
+    deploymentTarget: "12.0"
+    settings:
+      GENERATE_INFOPLIST_FILE: YES
+" >>project.yml
+
+xcodegen generate
 
 # Create a Podfile with our pod as dependency.
 echo "
-platform :tvos, '11.0'
+platform :tvos, '12.0'
 target '$PROJECT_NAME' do
   use_frameworks!
-  pod 'AEPCore', '~> 4.0'
-  pod 'AEPIdentity', '~> 4.0'
-  pod 'AEPServices', '~> 4.0'
-  pod 'AEPRulesEngine', '~> 4.0'
-  pod 'AEPEdge', '~> 4.0'
+  pod 'AEPCore', '~> 5.0'
```

Comment:
Should we use ~> 5.0.0?

Reply:
Using the optimistic operator here so `~> 5.0` means 5.0 and up until 6.0, but not including 6.0. So 5.0.0 is included.